unlock-protocol.com/blog/digital-cash.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Digital Cash. Big Deal.
+subTitle: Wallets and their superpowers for users.
+authorName: Sascha Mombartz
+publishDate: June 24, 2019
+description: A quick exploration into blockchain wallets, their applications and why they're so powerful.
+image: /static/images/blog/digital-cash/digitalcash.jpg
+---
+
+The wallet. It’s your interface into the blockchain. It lets you transact with the blockchain and in Ethereum’s case also with its smart contracts. You could think of it as an account but its more of an identifier with the crucial difference that you are in possession of it. It’s not stored on a server but lives with you, on your computer, your phone or hardware wallet.
+
+# Control & Transparency
+
+The fact that you’re in full control and the sole owner of your wallet and therefore the means with which you transact with the blockchain are a powerful aspect. 
+
+Think about it: your bank  account is managed by your bank, the data associated with it (your personal data, but also the data that represents your money) lives on servers that the bank controls, even your credit card is owned by the bank and governed by a cardholder agreement. It’s your money though! And it doesn’t end there. When your sending money your transaction is facilitated by the sending and receiving bank, plus the processor (Visa, Mastercard, etc.) and whoever else is in  the middle of that transaction (Western Union, Transferwise, Square, Stripe). And then you wonder, where is my money that’s mentioned in my bank account actually physically? The reason this matters is that you are dependent on the bank, their rules, fees and whims. If you take this to an extreme and imagine the bank going bankrupt, because they took on too much risk, you have no control and your money might be gone.
```

Comment:
Typo: "when you're sending"

Reply:
You could also mention that banks (or other intermediaries) have previously banned other companies or organizations, and not just the "illegal" ones, but all of the ones that they thought did not abide by _their_ terms of service (Wikileaks comes to mind... but also many crypto/blockchain companies)